peers.json

```diff
@@ -1,5 +1,5 @@
 {
-  "count": 4,
+  "count": 6,
```

Comment:
What reads this? Seems like `{peers}.uri.length` would be better to read?

Reply:
I don't know, maybe no one, maybe many as this file is public: http://myhush.org/config-generator/peers.json

I just noticed that last time I changed the nodes, I didn't update the 'count'.